addons/mail/static/tests/message/message.test.js

```diff
@@ -1960,7 +1934,7 @@ test("chatter - font size unchanged when there is only emoji", async () => {
     );
 });
 
-test("Copy Message Link", async () => {
+test.skip("Copy Message Link", async () => {
```

Comment:
not working (yet)?

Reply:
it works but not in testing. that's why i skipped it and didn't want to wast more time on this. I guess it is a problem of mocking `press()` but since it is only used here so i skiped

Reply:
Maybe need to click + trigger hot key instead?
We'd prefer `press()` as this is the HOOT way, but if the click + trigger hot key works I would be content to use them instead and not skip the test.